core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -32,6 +34,7 @@
  */
 @NotThreadSafe
 public abstract class Inode<T> implements JournalEntryRepresentable {
+  private static final Logger LOG = LoggerFactory.getLogger(Inode.class);
```

Comment:
Isn't it a good practice to have a logger defined in each class for the convenience of debugging?